python/python/bystro/ancestry/preprocess_1kgp_using_gnomad_loadings.sh

```diff
@@ -0,0 +1,37 @@
+#Extract gnomad loadings variant list from 1kgp genomes using plink
+
+#TODO convert this process to one using our own vcf parser
+
+#Download plink2 - can be downloaded with example below or at https://www.cog-genomics.org/plink/2.0/
```

Comment:
The user won't necessarily see this comment if they try to run the script without plink2 installed-- we could write something like:

```bash
# Define the program name
program_name="plink2"

# Check if the program is installed and present on $PATH
if ! command -v "$program_name" &> /dev/null; then
    echo "Error: '$program_name' is not installed or not present on the system's PATH.
    echo "Please install '$program_name' or make sure it's added to the PATH.  You can do this by blah blah blah"
    exit 1
fi

echo "'$program_name' is installed and present on the system's PATH."
```